### ISSUE TITLE
Changing my password on Crunchyroll causes the plugin to stop functioning

### ISSUE THREAD

**Issue by yogsodoth** [Yoshiofthewire/CrunchyXBMC#40]
Once I change my password on Crunchyroll, I just get a "Crunchyroll error" notification when I try to browse my queue through Kodi. Removing and re-installing the plugin doesn't seem to fix the issue.

**Comment by Yoshiofthewire** [comment]
Did you change the password in the plugin?

**Comment by yogsodoth** [comment]
Yep. Password changed in the plugin, still won't allow me to browse my queue.
Update: Manually deleted my .pickle file, then rebooted, the connection then worked.

**Comment by Yoshiofthewire** [comment]
The issue was with the session, the plugin is not seeing the password change
Probably a session issue
